notebooks/maternal_health_classification.ipynb

```diff
@@ -198,21 +411,431 @@
     "## EDA"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "cdad1333-e704-4eb9-8117-37daed73000f",
+   "metadata": {},
+   "source": [
+    "We can see that we don't have any missing values and also all features beside the target variable are numeric. "
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "0e9aae60-6101-4e5d-9e4c-7706f7162326",
+   "execution_count": 9,
+   "id": "be0e1267-e511-41d8-a7cc-57b27dc22397",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "import altair as alt"
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
+   "id": "e37a41be-7b65-448a-9f42-ce49a1c0ac74",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Age            0\n",
+       "SystolicBP     0\n",
+       "DiastolicBP    0\n",
+       "BS             0\n",
+       "BodyTemp       0\n",
+       "HeartRate      0\n",
+       "RiskLevel      0\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_eda.isnull().sum()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42cf25bd-aa6e-4e61-b2d5-d4b8905a9c8c",
+   "metadata": {},
+   "source": [
+    "We don't have too many observations (1014), but can still be enough for initial modeling"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "ad9c5fef-3851-40b6-8464-31c089c5a309",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(1014, 7)"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_eda.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d243717-b0e1-42e6-89d3-6106317200bf",
+   "metadata": {},
+   "source": [
+    "We have somewhat balanced data, with high risk having the fewest observations and low risk having the most."
```

Comment:
I think since we don't know if this would fall within balanced or unbalanced we could state the facts, the low risk class has the highest number of observations, while the high risk class has fewer observations, etc.